code/modules/mob/living/carbon/human/life.dm

```diff
@@ -397,6 +405,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 		if(drunkenness >= 61)
 			if(prob(50))
 				blur_eyes(5)
+			if(has_trait(TRAIT_DRUNK_HEALING) && prob(70))
+				adjustBruteLoss(-0.75, 0)
+				adjustFireLoss(-0.5, 0)
```

Comment:
Remove the prob(70) and change the healing to `-0.525` and `-0.35`, respectively